impl/src/planning.rs

```diff
@@ -74,6 +76,8 @@ struct DependencySet {
   build_deps: Vec<BuildableDependency>,
   // Dependencies that are required for tests
   dev_deps: Vec<BuildableDependency>,
+  // Dependencies that have been renamed and need to be aliased in the build rule
```

Comment:
Optional: This appears to contain strings of keys and values. This isn't obvious from the type Consider adding a note, like

```
  // Dependencies that have been renamed and need to be aliased in the build rule
  // formatted as `"$TARGET_NAME": "$BUILD_ALIAS"`
```

A slightly more type safe way to do this would be to use a simple struct containing these values, and do this formatting "later", but I don't feel strongly enough about this to really suggest it here.

Reply:
Just pushed a new commit pulling this out to a simple struct. Let me know what you think.

Reply:
This looks good to me, thanks for making this change. Marking resolved.